frontend/public/components/utils/_name-value-editor.scss

```diff
@@ -43,11 +49,11 @@
 
 .pairs-list__side-btn {
   cursor: pointer;
-  padding-right: 8px;
 }
 
 .pairs-list__action-icon {
-  padding-right: 0;
+  color: $color-pf-black-900;
+  margin-left: 8px;
```

Comment:
Would love to see the value as a var so it does not have to be repeated below.

Reply:
The other one is negative. I can make them both variables if you want, but they wouldn't be reused.

Reply:
You can make a positive var negative.  e.g., `-($var)`